plugins/bulk-import/src/components/PreviewFile/PreviewPullRequest.tsx

```diff
@@ -12,13 +12,13 @@ import {
 } from '@backstage/plugin-catalog-react';
 
 import { makeStyles } from '@material-ui/core';
-import TextField from '@material-ui/core/TextField';
 import Autocomplete from '@mui/material/Autocomplete';
 import Box from '@mui/material/Box';
 import Checkbox from '@mui/material/Checkbox';
 import CircularProgress from '@mui/material/CircularProgress';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import FormHelperText from '@mui/material/FormHelperText';
+import TextField from '@mui/material/TextField';
```

Comment:
The MUI4 text field was used to resolve the issue that occurred when resizing the screen while the preview side panel was open.

![Screenshot 2024-06-25 at 5 23 01 PM](https://github.com/janus-idp/backstage-plugins/assets/22490998/29652d7a-2760-47cf-9035-ad1f94113d0e)